src/components/app/data/services/programs.js

```diff
@@ -45,10 +45,10 @@ const retrieveCustomerContainsContent = async (enterpriseUuid, programUuid, cour
 };
 
 export async function fetchProgramDetails(enterpriseUuid, programUuid) {
-  const { DISCOVERY_API_BASE_URL, USE_API_CACHE } = getConfig();
+  const { DISCOVERY_API_BASE_URL } = getConfig();
   const url = `${DISCOVERY_API_BASE_URL}/api/v1/programs/${programUuid}/`;
   try {
-    const programResponse = await getAuthenticatedHttpClient({ useCache: USE_API_CACHE }).get(url);
+    const programResponse = await getAuthenticatedHttpClient().get(url);
```

Comment:
Curious, is this service call used with a React Query queryFn today?

Similar question for the other couple spots useCache is used today.

Reply:
Its currently used in conjunction [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/2e8d10ae0c0cdf1a2e7bc6950551cfa4c47d6dfa/src/components/app/data/services/programs.js#L51) when we fetch programs. There didnt seem to be any issue in retrieving programs when I verified the calls locally after the upgrade of frontend-platform to `8.1.5` which includes the upgrade to Axios to v1+